lib/maropost/do_not_mail_list.rb

```
@@ -0,0 +1,49 @@
+module Maropost
+  class DoNotMailList
+    def self.exists?(contact)
+      response = request(:get,
+                         "#{Maropost.configuration.api_url}/global_unsubscribes/email.json?contact[email]=#{contact.email}")
+      JSON.parse response.body
+
+      response['id'].present? ? true : false
+    rescue RestClient::ResourceNotFound, RestClient::BadRequest, RestClient::InternalServerError => e
+      contact.errors << 'Unexpected error occured'
```

Instead of `unepected error occured`, what if we use `e.message` for the error message? It may help debug a little quicker.

---

Also, should we be rescuing from `RestClient::InternalServerErrors` for all of these API requests? 

We use Rollbar to notify us of unhandled exceptions. If we're swallowing maropost 500 exceptions, we'll never get notifications when their API breaks (assuming 400/422 responses are common for invalid input).